src/AddExercise.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { getDatabase, push, ref} from 'firebase/database';
+import firebase from './firebase';
+
+const AddExercise = () => {
+
+    const [userInput, setUserInput] = useState("");
+
+    const handleInputChange = (e) => {
+        setUserInput(e.target.value)
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const database = getDatabase(firebase);
+        const dbRef = ref(database);
+        push(dbRef, userInput);
+        setUserInput("");
+    }
+
+return(
+    <section>
+        <div className="wrapper">
+            <form action="submit" className="addLiftForm">
+                <label htmlFor="exerciseChosen">Add an exercise to create your Workout</label>
+                <input type="text" id="exerciseChosen" onChange={handleInputChange} value={userInput} className="addLiftInput" name="exercise" placeholder="Enter Your Exercise" maxlength="24" required="true"/>
```

Comment:
UX: maybe add an alert to tell the user it's limited to 24 chars either by just adding a text line showing it or some other method, because currently it's not clear how it ends when the user is typing